includes/api/class-wc-rest-products-controller.php

```diff
@@ -608,6 +608,7 @@ protected function get_product_data( $product, $context = 'view' ) {
 			'manage_stock'          => $product->managing_stock(),
 			'stock_quantity'        => $product->get_stock_quantity( $context ),
 			'in_stock'              => $product->is_in_stock(),
+			'low_stock_amount'      => $product->get_low_stock_amount( $context ),
```

Comment:
This can't be introduced in this REST API, need to remove it and introduce only in v3 that are under development. All changes in this file need to be removed.